src/pages/main/index.tsx

```diff
@@ -19,7 +51,16 @@ const Main = () => {
           <Welcome />
         </div>
       </div>
-      <div className={styles.rightContainer}> Map</div>
+      <section aria-label="Map" className={styles.rightContainer}>
+        <Wrapper apiKey={import.meta.env.VITE_GOOGLE_MAPS_API_KEY}>
+          <Map center={mapCenter} onClick={onClick}>
+            <Marker position={mapCenter} icon={UserLocation} />
+            {targets.map((latLng, i) => (
+              <Marker key={i} position={latLng} />
```

Comment:
Same comment for the `key` here. Perhaps you could use something like `${lat}-${lng}`

Reply:
Fixed!